packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -52,6 +53,7 @@ export interface CompatConfig {
 	 * (Same version will be used across all layers).
 	 */
 	loadWith?: CompatVersion;
+	loadVersion?: string;
```

Comment:
As per my other comment you can remove this and generate this from `loadWith`. If you decide to keep it, add a comment here similar to the ones for createWith and loadWith which says this is for cross version compat only and that this is basically a resolved version from loadWith.